_posts/2024-04-18-bio-formats-7-3-0.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Release of Bio-Formats 7.3.0
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 7.3.0
+---
+
+Today we are releasing Bio-Formats 7.3.0 which includes compatibility and support for Bio-Formats with Java-21 alongside a number of format improvements including updates for DICOM, CellSens VSI, and PerkinElmer Columbus.
+
+The full list of changes included in this release are as follows:
+
+File format fixes and improvements:
+
+* CellSens VSI
+   - fixed and exception in CellSens 4.1 data by updating the handling of orphaned ets files (thanks to Edward Scanlon)
+
+* DICOM
+   - updated handling of nested sequences that include pixel data
+   - removed a duplicate enum check in the DICOM writer
+
+* OME-XML
+   - reading an OME-XML file now populates the original metadata table
+
+* PerkinElmer Columbus
+   - added support for reading data with Z stacks (thanks to Tim Blackmore)
+
+* Prairie Technologies TIFF
+   - improved type checking for Prairie files
+
+* SimplePCI
+   - fixed handling of Z position metadata
+
+Bio-Formats improvements:
+
+* Bio-Formats is now compatible with Java 21 builds
+* added a new `quality` option to `bfconvert` which enables the setting of the compression quality to be used. 
+* when using the `channel` option in `bfconvert`, the unselected channels are now correctly removed from the OME-XML metadata
+* added coverage of Apple M1 as part of the CI process
+* added coverage of Java 21 builds as part of the CI process
+
+Component updates:
+
+* `ome-jai` was upgraded to 0.1.4
+* `ome-stubs` was upgraded to 6.0.2
+* `ome-mdbtools` was upgraded to 5.3.3
+* `ome-common` was upgraded to 6.0.22
+* `ome-poi` was upgraded to 5.3.9
+* `ome-metakit` was upgraded to 5.3.7
+* `ome-codecs` was upgraded to 1.0.3
+* `ome-xml` was upgraded to 6.3.6
+* `specification` was upgraded to 6.3.6
```

Comment:
As with https://github.com/ome/bio-formats-documentation/pull/372#pullrequestreview-2006627066, there are some mismatches between the versions actually used in https://github.com/ome/bioformats/pull/4183 and what is here.